i18n/en.json

```diff
@@ -26,6 +26,7 @@
     "JavaScript is required for %s to work. Sorry for the inconvenience.": "JavaScript is required for %s to work. Sorry for the inconvenience.",
     "%s requires a modern browser to work.": "%s requires a modern browser to work.",
     "New": "New",
+    "Start over": "Start over",
```

Comment:
Two things:
- not required, but we did usually add new translations to the end of the file, so the untranslated new items are easy to find for translators
- you would need to add the new message (with English translation) to all files for the tests to work - this prevents exceptions at runtime due to missing translations and prevents the files getting out of sync. I usually use the jq tool to append new items to JSON files and ensure they are all formatted the same.